README.md

```diff
@@ -236,29 +236,26 @@ It is highly recommended to use a relational database cloud service like Amazon
 
 Follow the [instructions](https://docs.docker.com/engine/install/ubuntu/) to install Docker on Linux (Ubuntu).
 
-Once Docker is up and running, start the database by running the below script:
+Once Docker is up and running, start the database by running the below script (specify a password for the postgres user, **keep the password safe!**):
 
-> The terminal will prompt for the `power_user` password to use. **Keep it safe!**
-
-```sh
-docker run -d --name postgres \
-    --restart always \
-    -e POSTGRES_USER=power_user \
-    -e POSTGRES_PASSWORD=$(read -s;echo $REPLY) \
-    -p 1994:5432 \
-    postgres:14
+```bash
+POSTGRES_PASSWORD=<YOUR PASSWORD GOES HERE> docker compose -f postgres/docker-compose.yml up --detach
 ```
 
+TBD: explain that this creates the `postgres` super use for you
+TBD: explain what this does
+TBD: explain how to get to the logs
+
 ### Power User creation
 
-Regardless of installation approach, the postgres server will require the `power_user` to be created as a **SUPERUSER**.
+Regardless of installation approach, the postgres server will require the `postgres` to be created as a **SUPERUSER**. (if you used the docker installation instructions, this was already done for you)
 
-By default the `power_user` account has access to all databases on the server. We do not share the `power_user` credentials with the instance. The instance has its own credentials and ability to manage its own database.
+By default the `postgres` account has access to all databases on the server. We do not share the `postgres` credentials with the instance. The instance has its own credentials and ability to manage its own database.
 
-> Provide your own, secure password for the `power_user`. **Keep it safe!**
+> Provide your own, secure password for the `postgres`. **Keep it safe!**
 
 ```sql
-CREATE USER "power_user" WITH
+CREATE USER "postgres" WITH
```

Comment:
@tahitihat  Is there a reason we went with a "power_user" username? I guess it's more clear, I'm just so used to the "postgres" convention that you get for free. Any objections if we switch to postgres?

Reply:
The existing setup instructions for Postgres were written by @Ideonvn but I imagine `power_user` was selected for consistency with the Zenysis internal codebase (I'm happy to link to documentation offline). Slight preference for maintaining this consistency but agree that there's value in following broader conventions.

Reply:
@Sybrand this was essentially a clone from the zenysis internal codebase. Initially I just kept it the same to ensure nothing is dependant on it, but since then I've seen no usage for it. Happy to change to `postgres` 👍

Reply:
Have proposed 'n PR on the Zenysis repo to this effect.